src/common/datetime/format_date_time_with_seconds.ts

```diff
@@ -0,0 +1,23 @@
+import fecha from "fecha";
+
+// Check for support of native locale string options
+function toLocaleStringSupportsOptions() {
+  try {
+    new Date().toLocaleString("i");
+  } catch (e) {
+    return e.name === "RangeError";
+  }
+  return false;
+}
+
+export default toLocaleStringSupportsOptions()
```

Comment:
We should not add new `export default`, instead use a named export `export const formatDateTimeWithSeconds = `